app/code/community/Paylike/Paymentmethod/Model/Paymentmethod.php

```diff
@@ -26,7 +26,7 @@ public function authorize(Varien_Object $payment, $amount)
         $paylike = new Paylike($this->getApiKey());
         $result = $paylike->transactions->fetch($payment->getTransactionId());
         if ($transaction = $result->transaction) {
-            if ($transaction->id != $payment->getTransactionId() || $transaction->amount != $amount * 100 || Mage::app()->getStore()->getCurrentCurrencyCode() != $transaction->currency) {
+            if ($transaction->amount != $amount * 100 || Mage::app()->getStore()->getCurrentCurrencyCode() != $transaction->currency) {
```

Comment:
> You need to compare a unique order reference from your records with something on the transactions custom field.

See for instance how the WooCommerce plugin adds the order reference: https://github.com/paylike/plugin-woocommerce/blob/master/woocommerce-gateway-paylike/includes/class-wc-gateway-paylike.php#L316